dev/lint-python

```diff
@@ -27,6 +27,12 @@ MINIMUM_PYCODESTYLE="2.4.0"
 
 SPHINX_BUILD="sphinx-build"
 
+# Koalas depends on PySpark
```

Comment:
why is this change needed? (lint depends on Spark?)

Reply:
Doc generation actually imports the package to peek attributes.

Currently, Koalas checks if Pyspark is importable or not when Koalas is imported.